src/utils/networkUtils.js

```diff
@@ -35,7 +38,7 @@ export const hasWeb3 = () => {
 	return window.web3;
 };
 
-const defaultNetwork = { name: 'MAINNET', networkId: '1' };
+export const defaultNetwork = { name: 'MAINNET', networkId: '1' };
```

Comment:
Should the defaultNetwork used in the `wallet` reducer as the initial value of `networkName` and `networkId`? This is why I exported it. just wanted to double check.